generics/messageConstants/endpoints.js

```diff
@@ -18,7 +18,10 @@ module.exports = {
   GET_PROJECT_TEMPLATE_LISTS: 'api/v1/project/templates/listByIds',
   DOWNLOADABLE_FILE_URL: 'api/v1/cloud-services/files/getDownloadableUrl',
   FIND_ENTITY_DOCUMENTS: '/v1/entities/find',
-	FIND_ENTITY_TYPE_DOCUMENTS: '/v1/entityTypes/find',
+  LIST_BY_ENTITY_TYPE: '/v1/entities/listByEntityType',
+  FIND_ENTITY_TYPE_DOCUMENTS :'/v1/entityTypes/find',
+  GET_SCHOOL_DATA: '/v1/entities/listByLocationIds',
+  USER_CONSENT_API: '/v1/user/consent/update',
```

Comment:
@borkarsaish65 where are we using this. currently we don't have user consent endpint!

Reply:
removed now @VISHNUDAS-tunerlabs